Assets/Tests/PlayMode/Utility/Extensions/AuthScopeFlagsExtensionsTests.cs

```diff
@@ -20,16 +20,13 @@
  * SOFTWARE.
  */
 
-using NUnit.Framework;
-
-namespace PlayEveryWare.EpicOnlineServices.Tests
+namespace PlayEveryWare.EpicOnlineServices.Tests.Utility.Extensions
 {
+    using NUnit.Framework;
```

Comment:
What is NUnit.Framework for?

Reply:
It is the unit test framework used within Unity.